src/defaultsettings.cpp

```diff
@@ -131,6 +131,7 @@ void set_default_settings(Settings *settings)
 	settings->setDefault("always_fly_fast", "true");
 	settings->setDefault("directional_colored_fog", "true");
 	settings->setDefault("tooltip_show_delay", "400");
+	settings->setDefault("tooltip_show_itemstring", "false");
```

Comment:
looks like a previous contributor was being silly, there's no need to set the default twice

Reply:
Should I set the default in src/defaultsettings.cpp or in builtin/settingtypes.txt?
I think most settings are defined in both.

Reply:
in both, but only once per file

Reply:
I totally missed that...
Should I remove the other tooltip_show_delay too?

Reply:
There are other duplicates as well

Reply:
Otherwise they'd have to be removed in a separate commit, so go on and clean up the mess :)